### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#964] [Galaxy Trivia] [answered]
```
1, 11, 21, 1211, 111221, 312211, 13112221, ?
```

**Comment by peterbaccaro** [comment]
The next number in sequence is **1113213211**.

Each successive number in the sequence is formed by looking at the preceding number and reading out the digits, grouping like digits together and saying the number of identical digits followed by the digit.

e.g. take the no 1,11,21
- 1 is read as "one one", and so the second number is 11.
- 11 is read as "two ones", so the third number is 21.
- 21 is read as "one two, one one", so the fourth number is 1211.

Take the last given no in the sequence 13112221 so we read as:
- one 1
- one 3
- two 1’s
- three 2’s
- one 1

So the next no in sequence is 1113213211.

**Comment by CelioAmaral** [comment, marked as answer]
Sequence questions are very tricky because they admit any answer. Check it out:
1, 2, 3...
What is the next number in the sequence?
Of course, the vast majority will answer '4', because intuitively, this sequence leads us to that, but if someone says that it's 19, that's also right, because for any sequence of numbers, there are an infinity of polynomials that pass through the points.

In the given example:
p(x) = 2.5x³ - 15x² + 28.5x - 15
p(1) = 1
p(2) = 2
p(3) = 3
p(4) = 19

Satisfying sequence 1, 2, 3, 19

In this sequence question, just as we intuitively want to put 4 in the sequence 1, 2, 3... the sequence 1, 11, 21, 1211, 111221, 312211, 13112221 the missing number is 1113213211, because this sequence reports the previous number saying how many times each number repeats.

1
11 -> It means that in the previous number there is '1' number '1'.
21 -> It means that in the previous number we have '2' numbers '1'
1211 -> It means that in the previous number we have '1' number '2' and '1' number '1'
111221 -> '1' number '1' then '1' number '2' then '2' number '1' together
312211 -> '3' numbers '1' together, then '2' numbers '2' then '1' number '1'
13112221 -> '1' number '3', '1' number '1', '2' number '2' and '2' number '1'.

and so on.

The sequence was 1, 11, 21, 1211, 111221, 312211, 13112221, 1113213211, 31131211131221, ...

So the missing number in the sequence is:
1113213211
